src/zenml/integrations/gcp/orchestrators/vertex_orchestrator.py

```diff
@@ -414,6 +438,23 @@ def _construct_kfp_pipeline() -> None:
                     ]
                     container_op.after(upstream_container_op)
 
+                # Set optional CPU, RAM and GPU constraints for the pipeline
+                if self.cpu_limit is not None:
+                    container_op = container_op.set_cpu_limit(self.cpu_limit)
```

Comment:
Were you able to run with integer values for the cpu limit? When I set --cpu-limit=1000m it works but when I run --cpu-limit=1, I get the following failure:

TypeError: expected string or bytes-like object

at this exact line in code.

Reply:
Fixed.